lagom10-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/ServiceLocatorModule.scala

```diff
@@ -14,7 +14,7 @@ import play.api.inject.{ Binding, Module }
 class ServiceLocatorModule extends Module {
 
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] =
-    if (environment.mode == Mode.Prod)
+    if (Env.isRunByConductR)
```

Comment:
When is `isRunByConductR` true? Particularly, if `environment.mode` is equal to `Mode.Dev`, can `isRunByConductR` be true? Just asking because I don't want https://github.com/lagom/lagom/issues/42 to resurface :-)